docs/UserGuide.md

```diff
@@ -384,13 +382,14 @@ The simplest way to analyze multiple repos in one go is to use the `--repos` par
 * Format : `java -jar RepoSense.jar --repos REPO_LIST` <br>
 * Example: `java -jar RepoSense.jar --repos https://github.com/reposense/RepoSense.git c:/myRepose/foo/bar` analyzes the two specified repos (one remote, one local) and generates one report containing details of both.
 
-Alternatively, you can use csv config files to customize the analysis as before while specifying multiple repos to analyze.
-* `repo-config.csv`: Add additional rows for the extra repos ([example](repo-config.csv))
-* `author-config.csv`: Add one row for each author in each repo you want to analyze
+Alternatively, you can use csv config files to further customize the analysis:
+* `repo-config.csv`: Add a new row, to analyze a new repo ([example](repo-config.csv)).
+* `author-config.csv`: Add a new row, to specific the authors to analyze for the target repo(s) ([example](author-config.csv)).
+* `group-config.csv`: Add a new row, to customize the grouping of files for the target repo(s) ([example](group-config.csv)).
 
 ## Quickstart RepoSense with Netlify
 
-Using Netlify, and a fork of this repo allow you to quickly get started online and enjoy a real time RepoSense report on your target repository. Further instructions can be found in this [guide](UsingNetlifyGuide.md).
+To enjoy a real time RepoSense report on your target repositories, all you'll need is Netlify and a fork of this repo. Follow [this guide] to get started.
```

Comment:
Minor edit: I think it's sufficient to say `all you need is ...` rather than `all you'll need is...` ?